src/Controller/Organization/OrganizationEditController.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Organization;
+
+use App\Entity\Organization;
+use App\Form\Type\OrganizationEntityType;
+use App\Repository\OrganizationRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+
+/**
+ * @Route("/new", name="app_organization_new", methods={"GET"})
```

Comment:
I think you should use something like `@Route("/new", name="app_organization_new", methods={"GET", "POST"})`, and `@Route("/edit/{id}", name="app_organization_update", methods={"GET", "POST"})`